services/thermal_mgr/thermal_mgr.c

```diff
@@ -41,20 +41,46 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 }
 
+/**
+ * @brief Sends event to thermal manager queue
+ * 
+ * @param event - Pointer to thermal manager event to send
+ * @return ERR_CODE_SUCCESS if the event is successfully sent
+ */
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  xQueueSend(thermalMgrQueueHandle, event, 0);
```

Comment:
- [ ] In general, you should be checking that pointers aren't null before use. So check that event isn't null
- [ ] Check that thermal mgr queue handle isn't null, returning invalid state
- [ ] You should be checking the return value of non-void functions for error codes